ArjunaJTS/trailmap/pom.xml

```diff
@@ -1,122 +1,248 @@
-                        <version>2.2.3</version>
-                    </dependency>
-                </dependencies>
-                <executions>
-                    <execution>
-                        <id>generate-sources</id>
-                        <phase>generate-sources</phase>
-                        <goals>
-                            <goal>generate</goal>
-                        </goals>
-                        <configuration>
-                            <compiler>jacorb</compiler>
-                            <sources>
-                                <source>
-                                    <includes>
-                                        <include>com/arjuna/demo/simple/Hello.idl</include>
-                                        <include>com/arjuna/demo/jts/remotebank/Bank.idl</include>
-                                        <include>com/arjuna/demo/jts/explicitremotebank/Bank.idl</include>
-                                        <include>com/arjuna/demo/jts/txojbank/Bank.idl</include>
-                                    </includes>
-                                    <additionalArguments>
-                                        <additionalArgument>-DORB_OTS_SUPPORT=1</additionalArgument>
-                                        <additionalArgument>-DORB_CURRENT_SUPPORT=1</additionalArgument>
-                                    </additionalArguments>
-                                </source>
-                            </sources>
-                        </configuration>
-                    </execution>
-                </executions>
-            </plugin>
-        </plugins>
-        <pluginManagement>
-          <plugins>
-            <!--This plugin's configuration is used to store Eclipse m2e settings only. It has no influence on the Maven build itself.-->
-            <plugin>
-              <groupId>org.eclipse.m2e</groupId>
-              <artifactId>lifecycle-mapping</artifactId>
-              <version>1.0.0</version>
-              <configuration>
-                <lifecycleMappingMetadata>
-                  <pluginExecutions>
-                    <pluginExecution>
-                      <pluginExecutionFilter>
-                        <groupId>org.codehaus.mojo</groupId>
-                        <artifactId>idlj-maven-plugin</artifactId>
-                        <versionRange>[1.2.1,)</versionRange>
-                        <goals>
-                          <goal>generate</goal>
-                        </goals>
-                      </pluginExecutionFilter>
-                      <action>
-                        <ignore></ignore>
-                      </action>
-                    </pluginExecution>
-                  </pluginExecutions>
-                </lifecycleMappingMetadata>
-              </configuration>
-            </plugin>
-          </plugins>
-        </pluginManagement>
-    </build>
-</project>
+  LLC, and others contributors as indicated by the @authors tag. All rights 
```

Comment:
@marcosgopen there is a lot of code formatting change here, please can you only include the changed parts?